components/mediation-ui/org.wso2.carbon.sequences.ui/src/main/java/org/wso2/carbon/sequences/ui/util/SequenceEditorHelper.java

```diff
@@ -460,4 +483,44 @@ private static String getEditorMode(EditorUIClientFactory factory) {
     public static String getEditorMode(HttpSession session) {
         return getEditorMode(getFactoryFrom(session));
     }
+
+    /**
+     * Check the validity of a given mediator to be shown in UI
+     *
+     * @param mediator AbstractListMediator instance to be checked
+     * @return true if valid, false otherwise
+     */
+    private static boolean isValidMediatorForUI(Mediator mediator) {
+        boolean validity = true;
```

Comment:
Can further simplify contents in this particular method body as, "return !(mediator instanceof CommentMediator)"